frontend/src/components/forms/FormField.tsx

```diff
@@ -69,9 +73,19 @@ function field<T extends FieldType>(
         ref.current.setCustomValidity(error.message);
       }
     }, [ref, error.message, innerRef]);
+    /* eslint-disable */
```

Comment:
Forgot to ask about this. What does this disable?

Reply:
Nothing anymore, I was using it while testing stuff out. I forgot it there. Thanks for pointing that out.

I've started to double check pushes with difftools, hopefully I won't forget these in the future.